packages/aws-cdk-lib/aws-ec2/lib/vpc-endpoint.ts

```diff
@@ -553,10 +554,10 @@ export class InterfaceVpcEndpointAwsService implements IInterfaceVpcEndpointServ
    */
   public readonly privateDnsDefault?: boolean = true;
 
-  constructor(name: string, prefix?: string, port?: number) {
-    const region = Lazy.uncachedString({
+  constructor(name: string, prefix?: string, port?: number, global = false) {
```

Comment:
actually, maybe we want this to be `props: InterfaceVpcEndpointProps` in case we need to add more in the future?

Reply:
Yeah, I mentioned it in the PR: 
> I would have preferred to switch to a single object for the optional constructor properties (prefix, port, and now global), but couldn't make a breaking change to a publicly accessible constructor

I can put `global` in a `props` object, but we'll still have `name, prefix?, port?, props?`. Might be better late than never?

Reply:
Yeah, exactly: `name, prefix?, port?, props?`. Especially since global has a default value, will make it difficult to change in the future.

Reply:
Done 👍